specs/waku/v2/waku-swap-accounting.md

```diff
@@ -20,17 +21,54 @@ authors: Oskar Thorén <oskar@status.im>
 
 # Abstract
 
-This specification provides a way to do accounting based on the SWAP model proposed in Swarm, as well as being inspired by Bittorrent's economic model for bandwidth incentives. This is used for pairwise accounting to get nodes to cooperate and provide useful service and/or exchange of funds.
+This specification outlines how we do accounting and settlement based on the provision and usage of resources, most immediately bandwidth usage and/or storing and retrieving of Waku message. This enables nodes to cooperate and efficiently share resources, and in the case of unequal nodes to settle the difference through a relaxed payment mechanism in the form of sending cheques.
 
 **Protocol identifier***: `/vac/waku/swap/2.0.0-alpha1`
 
 # Motivation
 
-The Waku network makes up a service network, and some nodes provide a useful service to other nodes. We want to account for that, laying the foundation for settlement.
+The Waku network makes up a service network, and some nodes provide a useful service to other nodes. We want to account for that, and when imbalances arise, settle this. The core of this approach has some theoretical backing in game theory, and variants of it have practically been proven to work in systems such as Bittorrent. The specific model use was developed by the Swarm project (previously part of Ethereum), and we re-use contracts that were written for this purpose.
 
-TODO: Fill in more
+By using a delayed payment mechanism in the form of cheques, a barter-like mechanism can arise, and nodes can decide on their own policy as opposed to be strictly tied to a specific payment scheme. Additionally, this delayed settlement eases requirements on the underlying network in terms of transaction speed or costs.
 
-TODO: Game theoretical foundation, Bittorrent, relaxed tit-for-tat
+Theoretically, nodes providing and using resources over a long, indefinite, period of time can be seen as a an iterated form of prisoner's dilemma. Specifically, and more intuitively, since we have a cost and benefit profile for each provision/usage (of Waku Message's, e.g.), and the pricing can be set such that mutual cooperation is incentivzed, this can be analyzed as a form of donations game.
+
+# Game Theory - Iterated prisoner's dilemma / donation game
```

Comment:
Just saw this subsection, this subsection can be used as the link re my preceding comments